packages/opencensus-core/src/stats/view.ts

```diff
@@ -21,14 +21,14 @@ import {Bucket as metricBucket, DistributionValue, LabelValue, Metric, MetricDes
 import {TagMap} from '../tags/tag-map';
 import {TagKey, TagValue} from '../tags/types';
 import {isValidTagKey} from '../tags/validation';
-
 import {BucketBoundaries} from './bucket-boundaries';
 import {MetricUtils} from './metric-utils';
 import {Recorder} from './recorder';
 import {AggregationData, AggregationType, Measure, Measurement, StatsExemplar, View} from './types';
 
 const RECORD_SEPARATOR = String.fromCharCode(30);
 
+/** Implementation for the Base View class. */
```

Comment:
Could this comment be more descriptive in terms of the role that `BaseView` plays?

Reply:
Done in https://github.com/census-instrumentation/opencensus-node/pull/457/commits/be531aec1aa4c6439875507245449ea04cd65450. Thanks